docs/source/usage/usage.rst

```diff
@@ -141,7 +139,7 @@ Products are what Vendors sell, and the central Object in What's Fresh.
 
 **Pre-requisites**
 
-Products require Preparations. Make sure all the possible preparations this Product can have are created first. If an Image or Story is going to be added, these objects should be created before adding the Product.
+Products require Preparations. Make sure all the possible preparations this Product can have are created first if not you can do so by creating one directly from the product's from itself. If an Image or Story is going to be added, these objects should be created before adding the Product.
```

Comment:
This entire section can be deleted, since products no longer have prereqs -- everything can be added from the page :)